### ISSUE TITLE
OCC commands not working

### ISSUE THREAD

**Issue by Bodenseematze** [interfasys/galleryplus#32]
Regarding to the wiki  [https://github.com/interfasys/galleryplus/wiki/Create-Thumbnails](https://github.com/interfasys/galleryplus/wiki/Create-Thumbnails)   there should be occ commands to create / delete the thumbnails - which would be a really great feature!

But when I try the commands, I always get "there ar no commands defined in the namespace."
Neither
`occ gallery:create-thumbnails name`
nor
`occ galleryplus:create-thumbnails name`
works...

Is this a planned feature for a future version?
Or sholud this work?

I use OC 8.2.2 with galleryplus 14.3.0

**Comment by oparoz** [comment]
That feature is not available yet. You need to use a special branch of Gallery.
I need to re-sync the wiki so that it picks up the changes upstream.

**Comment by Bodenseematze** [comment]
What a pity - would be a real useful feature for me !

So will "galleryplus" in future versions of OC be "gallery"?

**Comment by oparoz** [comment]
Development is mostly done on Gallery, but as soon as new features are released, they're made available to older version of ownCloud.
That particular feature hasn't been released yet, but it should be in a month or so. There are not enough sponsors to pay for the development of all features in parallel, so things are done one at a time.

**Comment by effemmeffe** [comment]
Just stumbled in this problem too and luckily before reporting I searched.
Any ETA, since the predicted month is long gone now?
No pressure, just asking.

I got this:

```
fmf@kodi:~$ sudo -u www-data php /var/www/owncloud/occ gallery:create-thumbnails --all --regenerate
[sudo] password for fmf:

  [Symfony\Component\Console\Exception\CommandNotFoundException]
  There are no commands defined in the "gallery" namespace.

fmf@kodi:~$

```

**Comment by oparoz** [comment]
This feature has been postponed and may be introduced in the core component of ownCloud and Nextcloud.

**Comment by effemmeffe** [comment]
Thanks.
Is there some official ETA on next versions somewhere to be read? Again, no pressure, just for information.

Plus: Nextcloud?

**Comment by oparoz** [comment]
ownCloud will try to do something for 9.2 and it could then be part of [Nextcloud 11](https://nextcloud.com/)

**Comment by effemmeffe** [comment]
Little OT:
What are the difference between owncloud and nextcloud?
Why should I choose one instead of the other?
I just discovered ownclound and installed it, should I investigate nextcloud or it's ok to stay with owncloud?

**Comment by oparoz** [comment]
> What are the difference between owncloud and nextcloud?

Philosophy.
* ownCloud offers an open core and enterprise only features. They own the code
* Nextcloud is full open

You should read this:
https://nextcloud.com/who-is-nextcloud/

I will focus on Nextcloud.

**Comment by effemmeffe** [comment]
Damn, I just finished to install owncloud... :(
And I don't see a nextcloud package for ubuntu.
Is there some documentation to switch between owncloud and nexcloud?

**Comment by oparoz** [comment]
>Is there some documentation to switch between owncloud and nexcloud?

I suggest you check here:
https://help.nextcloud.com/

**Comment by effemmeffe** [comment]
Thanks.

**Comment by oparoz** [comment]
This issue was moved to oparoz/galleryplus#4
